numpy/_core/tests/test_strings.py

```diff
@@ -617,6 +672,62 @@ def test_replace_unicode(self, dt):
         assert_array_equal(np.strings.replace(buf,  "<", "&lt;", MAX),
                            "...\u043c......&lt;")
 
+    @pytest.mark.parametrize("dt", ["U", "T"])
+    @pytest.mark.parametrize("in_,out", [
+        ('\u1FFc', False),
+        ('\u2167', False),
+        ('\U00010401', False),
+        ('\U00010427', False),
+        ('\U0001F40D', False),
+        ('\U0001F46F', False),
+        ('\u2177', True),
+        ('\U00010429', True),
+        ('\U0001044E', True),
+    ])
+    def test_islower_unicode(self, in_, out, dt):
+        # TODO: Remove this
+        if dt == "T":
+            pytest.xfail(
+                "StringDType support to be added in a follow-up commit")
+        assert_array_equal(np.strings.islower(in_), out)
+
+    @pytest.mark.parametrize("dt", ["U", "T"])
+    @pytest.mark.parametrize("in_,out", [
+        ('\u1FFc', False),
+        ('\u2167', True),
+        ('\U00010401', True),
+        ('\U00010427', True),
+        ('\U0001F40D', False),
+        ('\U0001F46F', False),
+        ('\u2177', False),
+        ('\U00010429', False),
+        ('\U0001044E', False),
+    ])
+    def test_isupper_unicode(self, in_, out, dt):
+        # TODO: Remove this
+        if dt == "T":
+            pytest.xfail(
+                "StringDType support to be added in a follow-up commit")
```

Comment:
`pytest.xfail` does not have the `strict` keyword like the `xfail` mark. Would it help if I created one more function for `StringDType` and marked that as `xfail` instead of adding the `T` to `parametrize`?